pyproject.toml

```diff
@@ -28,6 +28,7 @@ depth-anything-v2 = { git = "https://github.com/landing-ai/depth-anything-v2.git
 controlnet-aux = { version = "^0.0.9", optional = true }
 lmdeploy = { version = "^0.5.3", optional = true }
 sam-2 = { git = "https://github.com/landing-ai/segment-anything-2.git", branch = "main", optional = true }
+diffusers = { git = "https://github.com/huggingface/diffusers.git", branch = "main", optional = true }
```

Comment:
QQ, why are we pointing to the dev version instead of the official version [available in pypi](https://pypi.org/project/diffusers/#:~:text=%F0%9F%A4%97%20Diffusers%20is%20the%20go-to%20library ) ?

Reply:
It was due to a previous local error but I will fix it, you are right